lesson-2/src/pages/layout/index.js

```diff
@@ -11,45 +11,36 @@ import Shop from '../shop/shop';
 import BookList from '../books/BookList';
 import BookShow from '../books/BookShow';
 import SignInPage from '../signin/signin';
-import { getUser, logoutUser } from '../../services/index';
+import { getUser, logoutUser } from '../../services';
+import PageNotFoundCompoment from '../static-pages/page-not-found';
 import './index.style.scss';
+import ProtectedRoute from './shared/ProtectedRoute';
```

Comment:
For consistency sake, you should change the file name `ProtectedRoute`  to  `protected-route`